lib/tools.js

```diff
@@ -105,8 +105,7 @@ function asJson (obj, num, time) {
         // this case explicity falls through to the next one
```

Comment:
This comment is no longer relevant.

Reply:
How so? It looks like it does explicitly fall through to the next case. I also don't see how it is relevant to this PR.

Reply:
I thought this comment was associated with the `'boolean'` case.

Reply:
Ahh, gotcha. Thanks giving a look!